app/main.ipynb

```diff
@@ -31,17 +31,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code for task 0 here"
+    "data_set = pd.read_csv(\"../data/adult.csv\")\n",
+    "data_set = data_set.replace(\"?\", np.nan)\n",
+    "data_set[\"salary K$\"] = np.where(data_set[\"salary K$\"] <= 0, np.nan, data_set[\"salary K$\"])\n",
```

Comment:
The logic for checking salary correctness seems to assume that 'salary K$' should be greater than 0. Ensure that this logic aligns with the dataset's context and task requirements.